src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -6122,5 +6122,136 @@ public void NestedShortTuplePatterns()
             }
             EOF();
         }
+
+        [Fact, WorkItem(32161, "https://github.com/dotnet/roslyn/issues/32161")]
+        public void ParenthesizedSwitchCase()
+        {
+            var text = @"
+switch (e)
+{
+    case (0): break;
+    case (-1): break;
+    case (+2): break;
+    case (~3): break;
+}
+";
```

Comment:
Are there other interesting cases? Perhaps:
```
    case (^1): break;
    case ((-1)): break;
```

Reply:
I don't think those cases are different in any way that is relevant to the bug or its fix.